src/main/java/org/jenkinsci/modules/windows_slave_installer/WindowsSlaveInstaller.java

```diff
@@ -124,9 +90,6 @@ public void install(LaunchConfiguration params, Prompter prompter) throws Instal
         final File agentExe = new File(dir, "jenkins-slave.exe");
         FileUtils.copyURLToFile(WindowsSlaveInstaller.class.getResource("jenkins-slave.exe"), agentExe);
 
-        FileUtils.copyURLToFile(WindowsSlaveInstaller.class.getResource("jenkins-slave.exe.config"),
```

Comment:
Should it also delete the existing one if it exists? AFAICT there should be no harm in the default configuration except non-surprising crashes on .NET Framework 2.0, but there should be no other risks. At the same time, there is a risk that user instances override the file.

Reply:
Agreed.